stream_alert_cli/runner.py

```diff
@@ -13,27 +13,27 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from stream_alert.shared import CLUSTERED_FUNCTIONS
 from stream_alert.shared.logger import get_logger
-from stream_alert_cli.apps.handler import app_handler
-from stream_alert_cli.athena.handler import athena_handler
+from stream_alert_cli.apps.handler import AppCommand
+from stream_alert_cli.athena.handler import AthenaCommand
 from stream_alert_cli.config import CLIConfig
-from stream_alert_cli.kinesis.handler import kinesis_handler
+from stream_alert_cli.configure.handler import ConfigureCommand
+from stream_alert_cli.kinesis.handler import KinesisCommand
 from stream_alert_cli.logger import set_logger_levels
-from stream_alert_cli.manage_lambda.deploy import deploy_handler
-from stream_alert_cli.manage_lambda.rollback import rollback_handler
-from stream_alert_cli.outputs.handler import output_handler
-from stream_alert_cli.rule_table import rule_staging_handler
-from stream_alert_cli.terraform.generate import terraform_generate_handler
-from stream_alert_cli.terraform.handlers import (
-    terraform_build_handler,
-    terraform_clean_handler,
-    terraform_destroy_handler,
-    terraform_init,
-    terraform_list_targets
-)
-from stream_alert_cli.test.handler import test_handler
-from stream_alert_cli.threat_intel_downloader.handler import threat_intel_downloader_handler
+from stream_alert_cli.manage_lambda.deploy import DeployCommand
+from stream_alert_cli.manage_lambda.rollback import RollbackCommand
+from stream_alert_cli.metrics_alarms.handler import CreateMetricAlarmCommand, \
+    CreateClusterMetricAlarmCommand, CustomMetricsCommand
+from stream_alert_cli.outputs.handler import OutputCommand
+from stream_alert_cli.rule_table import RuleStagingCommand
+from stream_alert_cli.status.handler import StatusCommand
+from stream_alert_cli.terraform.generate import TerraformGenerateCommand
+from stream_alert_cli.terraform.handlers import TerraformCleanCommand, TerraformBuildCommand, \
```

Comment:
same as above

Reply:
for reasoning behind this nit: http://google.github.io/styleguide/pyguide.html#32-line-length

specifically: `Do not use backslash line continuation except for with statements requiring three or more context managers.`